content/en/dashboards/graph_insights/correlations.md

```diff
@@ -38,17 +39,15 @@ Correlations *tries* to automatically detect the area of interest (anomalous beh
 
 **Note**: Correlation searches are available for a single metric. For graphs with multiple metrics, select the series of interest. From a full-screen graph, select one series on the graph legend, then click the **Correlations** tab.
 
-### Edit
+## Customize your search
 
 You can customize the default search parameters of correlations. From a full-screen graph, on the *Correlations* tab, click the **Edit Search** button, or click directly on the graph.
 
-* Click and drag on the graph to set the timeframe for your correlations search. If an area is already selected (pink box), you can move or resize the selection.
-* Define the sources you want correlations to search from (APM services, integrations, dashboards, or custom metrics).
-* `Auto-select` or `Custom select` from specific categories. For custom metrics, at least one selection is required.
-* Custom metrics is the only category not selected by default. Choose metric namespaces or single metrics to search correlations upon.
-* Use the tag filter box to scope the search by a tag.
-
-### Results
+1. Click and drag on the graph to set the timeframe for your correlations search. 
+1. Define the sources you want correlations to search from (APM services, integrations, dashboards, or custom metrics).
+1. Select `Auto-select` or `Custom select` from specific categories. For custom metrics, at least one selection is required.
```

Comment:
Should `Auto-select` or `Custom select` be bold instead of in code?

Reply:
Great catch
```suggestion
1. Select **Auto-select** or **Custom select** from specific categories. For custom metrics, at least one selection is required.
```